introduction/faq.md

```diff
@@ -19,23 +19,24 @@ title: Frequently asked questions (FAQ)
+
+### Is Qubes really free?
+`answer:` Qubes is free and open-source software. This means that everyone is free to use, copy, and change the software in any way. "Free" speaks to copyright and ownership concepts, and the freedom for any person to do anything they desire with source code. Qubes is also free of charge to anyone that wants to download and use it, and no matter how they wish to use it. Invisible Things Lab GmbH is the business entity that maintains and licenses Qubes OS. Qubes OS is published [under the GPLv2 license](https://www.qubes-os.org/doc/license/).
+
+### Is it safe for code to be so open like that? What about bad actors?
+The source code for Qubes is openly available so that anyone can contribute to and audit it. Only the CTO of Invisible Things Lab has the _[signing key](https://en.wikipedia.org/wiki/Code_signing)_ to create certified packages for the public to download and install. Signing code is a normal process that most organizations do, to garuntee the integrity of what users install on their machines. So, don't worry—it is a good thing that "just anyone" can audit and contribute to Qubes OS! Qubes is only possible, in fact, because of the large and generous community that contributes to building, auditing, and continually improving it.
 
 ### Why is OS security important?
 
-Most people use an operating system like Windows or macOS on their desktop and laptop computers.
-These OSes are popular because they tend to be easy to use and usually come pre-installed on the computers people buy.
-However, they present problems when it comes to security.
-For example, you might open an innocent-looking email attachment or website, not realizing that you're actually allowing malware (malicious software) to run on your computer.
-Depending on what kind of malware it is, it might do anything from showing you unwanted advertisements to logging your keystrokes to taking over your entire computer.
-This could jeopardize all the information stored on or accessed by this computer, such as health records, confidential communications, or thoughts written in a private journal.
-Malware can also interfere with the activities you perform with your computer.
-For example, if you use your computer to conduct financial transactions, the malware might allow its creator to make fraudulent transactions in your name.
+Most people use an operating system like Windows or macOS on their desktop and laptop computers. These OSes are popular because they tend to be easy to use and usually come pre-installed on the computers people buy. However, they present problems when it comes to security. Most common, anybody can open an innocent-looking file, email attachment, or website, not realizing that it's actually allowing malware—a catch-all phrase for any/all software created for malicious use, unbeknownst to an end user—to run on their computer.
```

Comment:
OSs, not OSes
"Most Common, anybody" - ? What about "For example, someone.."
"any/all" = "any"
Space either side of the hyphen.
There should be a hyphen after "use" instead of a comma, or put that in parentheses.
"unbeknownst to an end user" is wrong.

Reply:
I'll rewrite all of this in my copy editing pass after the substantive content matters have been worked out.

Reply:
@unman "For example" does not make clear why something is important, in an ordering of priorities. FAQs exist to deliver a TL;DR explainer to folks, and with infosec its important to clearly communicate that we're not willy-nilly making suggestions because they align with willy-nilly interests of ours, but because things really do happen that often. 

All of my recommendations are based on many years of user interviews in enterprise UX, that include dispelling IT myths end-users believe, because how IT likes to communicate often doesn't make sense to them. Some of my recommendations, are directly from interviewing journalists to learn about their security habits/beliefs. 

Hyphen was supposed to be an em-dash. 

Unsure how "unbeknownst to an end user" is incorrect? Malware doesn't ask for permission. It does things w/o consent. That is what I'm trying to communicate; that the user won't see what's happening.

Reply:
@andrewdavidwong "I'll rewrite all of this;" all of what I've written, or all of the mechanical language things Unman is pointing out? Unclear what, specifically, you're referring to. If my writing improvements just aren't desired at all, I'd prefer not to spend the time, here, doing them.